js/dcf-tabs.js

```diff
@@ -0,0 +1,124 @@
+class DCFTabs {
+  constructor(tabGroups) {
+    this.tabGroups = tabGroups;
+  }
+
+  initialize() {
+    Array.prototype.forEach.call(this.tabGroups, (tabGroup) => {
+      // Define constants for tabs
+      const tabList = tabGroup.querySelector('.dcf-tabs > ol, .dcf-tabs > ul');
+      const tabs = tabList.querySelectorAll('a');
+      const panels = tabGroup.querySelectorAll('.dcf-tabs > div, .dcf-tabs > section');
+      const uuid = DCFUtility.uuidv4();
+
+      // Prefix each tab group with a unique ID.
+      tabGroup.setAttribute('id', uuid.concat('-tab-group'));
+
+      // Tab switching function
+      const switchTab = (oldTab, newTab) => {
```

Comment:
update function definition to `tabGroup.prototype.switchTab = (oldTab, newTab) => {` and update the calls to the method to `this.switchTab`

Reply:
I had to use this.tabGroups.switchTab when calling the method. Is this okay?

Reply:
That's fine if it works.  I didn't test my suggestion beyond gulp lint.